net-misc/trackma/trackma-0.7.3.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_4,3_5} )
+
+inherit eutils distutils-r1
+
+SRC_URI="https://github.com/z411/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+DESCRIPTION="Open multi-site list manager for media tracking sites"
+HOMEPAGE="https://github.com/z411/trackma"
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="inotify cli +urwid gtk qt5"
+
+DEPEND="urwid? ( dev-python/urwid[${PYTHON_USEDEP}] )
+	gtk? (
+		dev-python/pygobject:*[${PYTHON_USEDEP}]
+		dev-python/pycairo[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+	)
+	qt5? (
+		dev-python/PyQt5[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+	)"
+RDEPEND="sys-process/lsof
+	inotify? ( || ( dev-python/inotifyx dev-python/pyinotify ) )
+	${DEPEND}"
+
+python_install_all() {
+	distutils-r1_python_install_all
+	newicon "${S}/trackma/data/icon.png" trackma.png
+	use cli && make_desktop_entry "${PN}" "${PN^} (cli)" "${PN}" "Network" "Terminal=true"
```

Comment:
I'm a bit confused. Are they all installed unconditionally but you have flags to enforce deps and create menu entries?